UnityProject/Assets/Scripts/Health/Electrocution.cs

```diff
@@ -1,162 +1,50 @@
-using System;
 using UnityEngine;
 
 /// <summary>
-/// For electrocuting a victim. NPCs not emplemented, just (human) players.
+/// The rated severity of an electrocution.
 /// </summary>
-public class Electrocution
+public enum ElectrocutionSeverity
 {
-	/// <summary>
-	/// The electrocution severity.
-	/// <para>None: no potential difference (no side effects)</para> 
-	/// <para>Mild: exposure to voltage like that of a small battery (no consequential effects)</para>
-	/// <para>Painful: imagine an electric fence (drop actively held item, minor burns)</para>
-	/// <para>Lethal: high voltage line (stun, severe burns, death possible)</para>
-	/// </summary>
-	public enum Severity
-	{
-		None,
-		Mild,
-		Painful,
-		Lethal
-	}
+	/// <summary> No potential difference (no side effects) </summary>
+	None,
+	/// <summary> Exposure to voltage like that of a small battery (no consequential effects) </summary>
+	Mild,
+	/// <summary> Exposure to high voltage - akin to an electric fence (drop actively held item, minor burns) </summary>
+	Painful,
+	/// <summary> Exposure to very high voltage (stun, severe burns, death possible) </summary>
+	Lethal
+}
 
-	private const int NON_INSULATED_ITEM_RESITANCE = 20000; // 20 kilo Ohms
+public class Electrocution
```

Comment:
If an electrocution is supposed to indicate an external exposure to electricity, then this object should be immutable and all information on the exposure should be present upon construction. There should be no ability or need to modify / set fields. 

Adding the ability to modify the details of the exposure by any implementation of IElectrocutable will add complexity / confusion to this process, and I don't see why it should be needed anyway...if you're exposing a bunch of things to electricity they shouldn't need to communicate between each other, other than knowing how strong the exposure is in terms of some quantitative value.